linux/user-and-file-management/manipulating-environment-variables/manipulating-environment-variables-ii.md

```diff
@@ -15,72 +9,79 @@ type: normal
 
 category: must-know
 
-inAlgoPool: false
-
 tags:
-
   - introduction
 
 
 ---
 
-# Manipulating Environment Variables (Part 2)
+# Updating & Deleting
 
 ---
 ## Content
 
-### Capitalizing Environment Variables
 
-First, by convention, environment variables are given names in ALL CAPS and underscores `_`.  It is possible, but not recommended, to use lower-case letters when naming environment variables. You'll only ever see us use ALL CAPS.
+### Modifying an Existing Variable
 
-### Displaying a Variable
+You'll often want to modify an existing variable by appending or prepending data. To do this, you can reference an environment variable inside a string:
 
-Use `echo` to print the contents of a variable.  If you reference a variable that hasn't been created yet it will have an empty string `""` as a default value.
-
-Here `HOME` contains the full path to the current user's home directory while `FOOD` has not been set.
-
-```shell
-enki@host ~ $ echo $HOME
-/home/enki
-enki@host ~ $ echo $FOOD
+```bash
+export FOOD="waffles"
+echo $FOOD
+waffles
 
-enki@host ~ $
+export FOOD="chocolate $FOOD"
+echo $FOOD
+chocolate waffles
 ```
 
-### Creating a New Variable
-
-You can create a new environment variable using the `=` operator and the `export` command, like so:
+### Unsetting an Environment Variable
 
-```shell
-enki@host ~ $ echo $FOOD
+To unset or clear an environment variable use `unset`:
 
-enki@host ~ $ export FOOD="waffles"
-enki@host ~ $ echo $FOOD
+```bash
+export FOOD="waffles"
+echo $FOOD
 waffles
-enki@host ~ $
-```
 
-Make sure not to add any spaces around `=`.
+unset FOOD
+echo $FOOD
 
-```shell
-enki@host ~ $ export FOOD = "waffles"
--bash: export: `=': not a valid identifier
-enki@host ~ $
 ```
 
+> If you reference a variable that hasn't been created yet (or that has been deleted), it will have an empty string `""` as a default value.

```

Comment:
```suggestion
> 💡If you reference a variable that hasn't been created yet (or that has been deleted), it will have an empty string `""` as a default value.

```